docs/guides/headless-cms/build-static-website/next.md

```diff
@@ -259,7 +259,7 @@ async function getPost(slug) {
 	try {
 		const post = await directus.request(
 			readItem('posts', slug, {
-				fields: ['*', { relation: ['*'] }],
+				fields: ['*.*', { relation: ['*'] }],
```

Comment:
Wrong syntax. Also, I think `relation` is the wrong key here - is it not `author`?

Reply:
The dot notation `*.*` is to resolve all relations. I've made an update to this.

Reply:
That isn't right as of the new SDK. It should be:
`fields: ['*', { '*': ['*'] }],`, but can we be more specific and just pull in the author name, as required?

Reply:
`fields: ['*', { '*': ['*'] }],` is not supported make sure to only use wildcards as the final item in the chain:
- `fields=*` -> `fields: ['*']
- `fields=test.*` -> `fields: [{ 'test': ['*']}]`
- `fields=*.*` -> not supported

Reply:
Well there we go. Everybody is wrong haha

Reply:
@br41nslug The syntax `*.*` still works. Should it still work when unsupported? 🤔  

I've updated the snippet based on what you sent above ^

Reply:
*not supported by the Typescript types, if you're not using typescript or are ignoring the error then the API still accepts it just fine

Reply:
Alright, this makes sense